README.md

```diff
@@ -65,6 +65,9 @@ Listing a few git resources here for everyone.
 * In my opinion this was a good step to conduct such interactive meetups to bring the like minded people at one place. It helps to share experiences and keeps up to date with the modern technology. We can learn and share knowledge to compete with developed countries and make our country an Advanced country. I think such meetups should be conducted twice or thrice in a month for different fields of interest. – Muhammad Ramzan
 * This event was realy informative motivational. I inspired a lot by the speaker behaviour and his sense of humour. I think this kind of should properly arranged on monthly basis this will a greate contribution from your side in national building. – Muhammad Ramzan
 
+<ul>
```

Comment:
This is not Markdown. Use Markdown. Update it. Read the [cheatsheet](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf) which you received at the Meetup.